### DISCUSSION TITLE
[feature request] equalizer

### DISCUSSION THREAD

**Discussion by int3ks** [PaulWoitaschek/Voice#1353] [Ideas]
hi. is it possible to add an equalizer for the app only? i use my phone in car with music player too. and for audiobooks there is way touch bass... if i lower bass in system panel it to low for music.
tia 
inteks ;o)

**Comment by Schninzent** [comment]
That would be such a great feature! Often Audiobooks are too high or too low in bass/treble. An option to change these would be the cherry on top.
